packages/openapi-typescript/test/transform/schema-object/array.test.ts

```diff
@@ -45,6 +45,72 @@ describe("transformSchemaObject > array", () => {
         want: `[
     number,
     number,
+    number,
+    ...number[]
```

Comment:
This other modification to an existing test may also _technically_ be a breaking change, but I think because it’s purely additive, could be considered a bugfix. I would be OK with this going in as-is.